integration-tests/oidc-wiremock-logout/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-quarkus.keycloak.devservices.enabled=false
+quarkus.devservices.enabled=false
```

Comment:
That appears to disable it completely for all platforms

Reply:
Never mind, looks like they are indeed not required for this test - though a more general fix in the linked seems to work better in a sense where we want to check if a lightweight OIDC service should be used, so that PR is needed anyway.